src/main/java/Duke.java

```diff
@@ -1,10 +1,107 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    private static final String MESSAGE_LINEBREAK = "____________________________________________________________";
+    private static final String MESSAGE_GREETING = MESSAGE_LINEBREAK + "\n" +
+            "Hello! I'm Duke\n" +
+            "What can I do for you?\n" +
+            MESSAGE_LINEBREAK;
+    private static final String MESSAGE_BYE = "Bye. Hope to see you again soon!";
+    private static final String MESSAGE_ERROR = "Please enter a valid command";
+    private static ArrayList<Task> taskList = new ArrayList<Task>();
+    private static Scanner in = new Scanner(System.in);
+    private static String inputCommand = "";
+    private static int numberOfTasks = 0;
+
+    public static String firstWordOf(String input){
+        if(input.contains(" "))
+            return inputCommand.substring(0,inputCommand.indexOf(' '));
+        else
+            return input;
```

Comment:
Remember to put curly brackets to wrap any number of statements in an if-else block. Even if it is just 1 statement.